backend/resolvers/queries/userQuery.js

```diff
@@ -3,7 +3,7 @@ import { UserError } from 'graphql-errors'
 import { STUDENT, ADMIN, JUDGE } from '../../constants'
 
 export function user (_, args, req) {
-  const isRequestingOwnUser = req.auth.id !== undefined && req.auth.id === args.id
+  const isRequestingOwnUser = req.auth.username !== undefined && req.auth.username === args.id
```

Comment:
seemed to be broken.... fixed a bit but the schema still expects `id` in the args, which we should think about

Reply:
Graphql IDs don't necessarily mean int ideas, just unique ids. The ID type is serialized in the same way as a String. So, I don't think I'd want to stop identifying the user's pk as a graphql id. I rather change the pk for user to a normal id.